src/components/Garden/Stake/StakingMovements.tsx

```diff
@@ -150,10 +158,11 @@ function StakingMovements({ stakingMovements, token }) {
           <div>
             <Tag
               background={
-                actionAttributes.background && `${actionAttributes.background}`
+                actionAttributes?.background &&
+                `${actionAttributes?.background}`
```

Comment:
That check its not necessary, because its return undefined if its not defined, so its redundant.

```suggestion
                actionAttributes?.background
```

Or keep the template string expression (back tick `)

Reply:
Makes sense. Will change it